data/team_alumni.yml

```diff
@@ -7,29 +7,54 @@ members  :
     ncaruname   : 
     githubuname : hCraker
 
+  - name        : ALINA GUHA
+    designation : SIParCS Intern
```

Comment:
Would it be useful to mention the year of internship (e.g. '22) for alumni?

Reply:
I just added in the full years for clarity.